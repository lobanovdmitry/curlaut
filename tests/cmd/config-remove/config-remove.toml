args = "config remove local"
fs.cwd = "fs"
fs.sandbox = true

[env.add]
HOME = "."
