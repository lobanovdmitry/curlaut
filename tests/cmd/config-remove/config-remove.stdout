Removing keycloak by alias local
